src/Fabulous.XamarinForms/Views/Controls/RadioButton.fs

```diff
@@ -0,0 +1,174 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Fabulous.XamarinForms
+open Xamarin.Forms
+
+type IRadioButton =
+    inherit ITemplatedView
+
+module RadioButton =
+    let WidgetKey = Widgets.register<RadioButton> ()
+
+    let BorderColor =
+        Attributes.defineAppThemeBindable<Color> RadioButton.BorderColorProperty
+
+    let GroupName =
+        Attributes.defineBindable<string> RadioButton.GroupNameProperty
+
+    let BorderWidth =
+        Attributes.defineBindable<float> RadioButton.BorderWidthProperty
+
+    let CharacterSpacing =
+        Attributes.defineBindable<float> RadioButton.CharacterSpacingProperty
+
+    let CornerRadius =
+        Attributes.defineBindable<float> RadioButton.CornerRadiusProperty
+
+    let Content =
+        Attributes.defineScalarWithConverter<Content.Value, Content.Value, Content.Value>
+            "RadioButton_Content"
+            id
+            id
+            ScalarAttributeComparers.equalityCompare
+            (fun newValueOpt node ->
+                let target = node.Target :?> BindableObject
+
+                match newValueOpt with
+                | ValueNone -> target.ClearValue(RadioButton.ContentProperty)
+                | ValueSome converter ->
+                    match converter with
+                    | Content.String string -> target.SetValue(RadioButton.ContentProperty, string)
+                    | Content.Widget widget ->
+                        let view =
+                            Helpers.createViewForWidget node widget |> unbox
```

Comment:
Hmm. This wouldn't work with diffing.
I guess I would create 2 attributes `ContentString` / `ContentWidget` and make 2 constructors accepting one or the other. That way `ContentString` will be a scalar property and `ContentWidget` will be a widget property supporting diffing

Reply:
Both would set the `RadioButton.ContentProperty`
It's not an issue because the only way to set those attributes is to use the corresponding constructor

Reply:
You don't need `Content.Value` anymore. `ContentString` will be `Attributes.defineBindable<string>` and `ContentWidget` will be `Attributes.defineBindableWidget`

Reply:
Opps . Yeah that is even better 😅

Reply:
Updated to use Attributes.defineBindableWidget